src/figuresClasses.ts

```diff
@@ -1,19 +1,73 @@
-export interface Figure {
 
+export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number,
 }
 
 export class Triangle {
+  shape = 'triangle';
+
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (
+      this.a + this.b <= this.c
+      || this.c + this.b <= this.a
+      || this.c + this.a <= this.b
+    ) {
+      throw new Error('Error');
+    }
+  }
 
+  getArea(): number {
+    const peremeter: number = (this.a + this.b + this.c) / 2;
+
+    // eslint-disable-next-line max-len
+    return Math.floor(Math.sqrt(peremeter * (peremeter - this.a) * (peremeter - this.b) * (peremeter - this.c)) * 100) / 100;
+  }
 }
 
 export class Circle {
+  shape = 'circle';
```

Comment:
```suggestion
  shape = 'circle';
```
Use enum instead of string